rules/windows/discovery_enumerating_domain_trusts_via_nltest.toml

```
@@ -71,16 +78,19 @@ process where host.os.type == "windows" and event.type == "start" and
     process.name : "nltest.exe" and process.args : (
         "/DCLIST:*", "/DCNAME:*", "/DSGET*",
         "/LSAQUERYFTI:*", "/PARENTDOMAIN",
-        "/DOMAIN_TRUSTS", "/BDC_QUERY:*")
+        "/DOMAIN_TRUSTS", "/BDC_QUERY:*"
+        ) and 
+not process.parent.name : "PDQInventoryScanner.exe" and not user.id : "S-1-5-18"
```

is this intended to exclude `PDQInventoryScanner.exe` by `S-1-5-18` user or are these intended to be 2 separate exclusions?

---

These are two separate exclusions.

---

just to make it more obvious 

```suggestion
  not process.parent.name : "PDQInventoryScanner.exe" and 
  not user.id : "S-1-5-18"
```